### ISSUE TITLE
Itemtransporters stop pulling from the heated centrifuge

### ISSUE THREAD

**Issue by NelDav** [FTBTeam/FTB-Modpack-Issues#6221]
### Modpack

FTB Evolution

### Modpack version

1.3.0

### Launcher

CurseForge

### Has the pack been modified

No

### Describe the bug

When using an item transporter (industrial foregoinge) to pull items from the heated centrifuge into the dimensional chest, the transporter will stop pulling after a full cycle to the 9 output slots. It will then start pulling again, if the input of the heated centrifuge is empty again. Then it will do a full cycle through all output slots again.

### Steps to reproduce

Set up a dimensional chest, place a heated centrifuge, pipe combs into the centrifuge (I suggest obsidian combs because they fill the output slots quite fast). Then use an item transporter to pull from the heated centrifuge into the dimensional chest. It is important that there are always combs in the input slot, otherwise you will not see the issue.

### Expected behaviour

The item transporter should only directly restart pulling from all output slots, when reaching the end of all output slots.

**Comment by Volpamonhub** [comment]
I don't think it's a bug, but rather the normal way the item transporter works. It cycles through each slot in order, even if a previous slot that it has already pulled from fills up again. 

**But I could be wrong.**

Anyway, you can 'fix' this issue by adding some upgrades to make the extraction faster, so it cycles through more quickly. Otherwise, you might have to switch to a different transport method.

**Comment by NelDav** [comment]
> Anyway, you can 'fix' this issue by adding some upgrades to make the extraction faster, so it cycles through more quickly.

No, the problem is, that it won't pull anything, until the input slot is empty.

I am not sure, but I think the reason for this is as follow:

1. The item transporter starts to pull from the first output slot.
2. As soon as this slot is empty, it checks the second output slot and starts pulling until this slot is empty
3. After checking each output slot, the transporter checks the input slot (this is already wrong, why would I want to pull the honey combs out of the centrifuge)
4. The centrifuge does not allow pulling items out of the input slot. Hence, the item transporter does not pull anything.
5. The item transporter waits until the centrifuge processed all combs.
6. Since all honey combs are processed, the input slot is empty and the item transporter starts checking the first output slot again.

The problem is, if there are so many combs, that all 9 output slots are occupied, the input slot won't become empty, and the whole setup is stuck.

**Comment by alfredggttv** [comment]
If I'm understanding this properly, this seems like a bug/issue with the Item Transporter itself, maybe even a limitation. That would be something for the mod dev to know about, and maybe they could clear the issue up.
